res/css/views/rooms/_SearchBar.pcss

```
@@ -15,30 +15,43 @@ limitations under the License.
 */
 
 .mx_SearchBar {
-    height: 56px;
+    min-height: 56px; /* for wrapped mx_SearchBar_button making the area tall */
```

more context in the comment please

```suggestion
    /* use `min-height` rather than height, to allow room for the text to wrap if the window is narrow */
    min-height: 56px;
```